### PR TITLE
Update README.zh-Hans.md

### PR DESCRIPTION
**Pull request by epiciskandar**
Fix misleading zoom instruction, which gave opposite action steps.

操作说明写反了，造成误导。

### CODE REVIEW

**Comment by rom1v** [comment]
cc @Win7GM @yume-chan

File: README.zh-Hans.md [hunk 1]
```diff
@@ -706,7 +706,7 @@ scrcpy --disable-screensaver
 
 模拟“双指缩放”：<kbd>Ctrl</kbd>+_按住并移动鼠标_。
 
-更准确的说，在按住鼠标左键时按住 <kbd>Ctrl</kbd>。直到松开鼠标左键，所有鼠标移动将以屏幕中心为原点，缩放或旋转内容 (如果应用支持)。
+更准确的说，在按住 <kbd>Ctrl</kbd> 时按下鼠标左键。直到松开鼠标左键，所有鼠标移动将以屏幕中心为原点，缩放或旋转内容 (如果应用支持)。
 
 实际上，_scrcpy_ 会在关于屏幕中心对称的位置上用“虚拟手指”发出触摸事件。
```
Comment (yume-chan) [inline README.zh-Hans.md:711]: I think this part can be phased better, for example something like

```diff
- 模拟“双指缩放”：<kbd>Ctrl</kbd>+_按住并移动鼠标_。
+ 模拟“双指缩放”：<kbd>Ctrl</kbd>+_按下并拖动鼠标_。

- 更准确的说，在按住 <kbd>Ctrl</kbd> 时按下鼠标左键。直到松开鼠标左键，所有鼠标移动将以屏幕中心为原点，缩放或旋转内容 (如果应用支持)。
+ 准确的说，按住 <kbd>Ctrl</kbd>，然后按下鼠标左键。在松开鼠标左键前，移动鼠标会使屏幕内容相对于屏幕中心缩放或旋转 (如果应用支持)。

- 实际上，_scrcpy_ 会在关于屏幕中心对称的位置上用“虚拟手指”发出触摸事件。
+ 具体来说，_scrcpy_ 会在鼠标位置关于屏幕中心对称的位置上再生成一个“虚拟手指”。
```

**Review by yume-chan — COMMENTED** [review]
The old translation is not like common Chinese, so I'd like the whole part to be improved.

File: README.zh-Hans.md (hunk 1 above)
Comment (Win7GM) [reply README.zh-Hans.md:711]: > I think this part can be phased better, for example something like
> 
> ```diff
> - 模拟“双指缩放”：<kbd>Ctrl</kbd>+_按住并移动鼠标_。
> + 模拟“双指缩放”：<kbd>Ctrl</kbd>+_按下并拖动鼠标_。
> 
> - 更准确的说，在按住 <kbd>Ctrl</kbd> 时按下鼠标左键。直到松开鼠标左键，所有鼠标移动将以屏幕中心为原点，缩放或旋转内容 (如果应用支持)。
> + 准确的说，按住 <kbd>Ctrl</kbd>，然后按下鼠标左键。在松开鼠标左键前，移动鼠标会使屏幕内容相对于屏幕中心缩放或旋转 (如果应用支持)。
> 
> - 实际上，_scrcpy_ 会在关于屏幕中心对称的位置上用“虚拟手指”发出触摸事件。
> + 具体来说，_scrcpy_ 会在鼠标位置关于屏幕中心对称的位置上再生成一个“虚拟手指”。
> ```

The last sentence will be easier to understand if changed into sth like:

具体来说，在与鼠标位置关于屏幕中心对称的位置上，_scrcpy_ 会生成一个“虚拟手指”用于模拟点按操作。

**Review by Win7GM — CHANGES_REQUESTED** [review]

File: README.zh-Hans.md (hunk 1 above)
Comment (yume-chan) [reply README.zh-Hans.md:711]: @Win7GM I want to emphasize that touch events are generated for BOTH mouse pointer and virtual finger, so I prefer adding a "再“.

Comment (Win7GM) [reply README.zh-Hans.md:711]: > @Win7GM I want to emphasize that touch events are generated for BOTH mouse cursor and virtual finger, so I prefer adding a "再“.

@yume-chan I think either way will do but if it should be emphasized I would prefer phrasing it explicitly like:
`具体来说，在鼠标，和与鼠标位置关于屏幕中心对称的位置上，_scrcpy_ 会分别生成一个“虚拟手指”用于模拟点按操作。`

Comment (yume-chan) [reply README.zh-Hans.md:711]: @Win7GM Now the meaning of "virtual finger" now differs from the original English text:

> Concretely, scrcpy generates additional touch events from a "virtual finger" at a location inverted through the center of the screen.

Although it's not unacceptable.

Comment (Win7GM) [reply README.zh-Hans.md:711]: @yume-chan I start to feel if we remove this “虚拟手指” from the translation it may fit the original meaning better.

Comment (yume-chan) [reply README.zh-Hans.md:711]: @Win7GM I guess that's also fine. Something like 

> 具体来说，Scrcpy 会在鼠标位置，以及鼠标以屏幕中心镜像的位置分别生成触摸事件。

Comment (Win7GM) [reply README.zh-Hans.md:711]: @yume-chan Agreed.

Comment (epiciskandar) [reply README.zh-Hans.md]: Hi guys, this is being updated, please check it and we can see if everything applicable now.

Comment (yume-chan) [reply README.zh-Hans.md]: @epiciskandar Should be

```markdown
模拟“双指缩放”：<kbd>Ctrl</kbd>+_按下并拖动鼠标_。

准确的说，按住 <kbd>Ctrl</kbd>，然后按下鼠标左键。在松开鼠标左键前，移动鼠标会使屏幕内容相对于屏幕中心缩放
或旋转 (如果应用支持)。

具体来说，Scrcpy 会在鼠标位置，以及鼠标以屏幕中心镜像的位置分别生成触摸事件。
```

I don't like both `准确的说` and `具体来说`, can you guys come up with something better?

Comment (Win7GM) [reply README.zh-Hans.md]: @epiciskandar @yume-chan I think `准确的说` can be removed.
About `具体来说` I think we can alter it into some sentences including sth like `实现原理`.

Comment (epiciskandar) [reply README.zh-Hans.md]: Might we stray too far from original English version?

Comment (Win7GM) [reply README.zh-Hans.md]: @epiciskandar I think this part will never be altered / will only be altered slightly in the English version and thus I choose to adopt free translation.
The meanings are consistent anyway.

Comment (epiciskandar) [reply README.zh-Hans.md]: OK, lets' forget unimportant disputes. How about now

Comment (yume-chan) [reply README.zh-Hans.md]: The second half of the second paragraph is still incorrect.

Comment (epiciskandar) [reply README.zh-Hans.md]: sorry @yume-chan, could you please make another conversation on the updated version? I can't figue out what needs improvements.

Comment (yume-chan) [reply README.zh-Hans.md]: @rom1v I did the old translation, but when I read it again now, some choices of words are pretty bad.

@epiciskandar's initial change is good, but I want to discuss and improve the whole section taking this opportunity.

@Win7GM and I mostly agreed on the above version, the problem now is @epiciskandar mixed the second and third paragraphs up.

So after changing that or maybe several more discussions after this PR will be good to go.

Comment (epiciskandar) [reply README.zh-Hans.md]: Docs are important, especially for non-programming skill people, it worth to be patient, and let's make it efficient too, correcting contents is more important than beautify wordings

**Review by Win7GM — APPROVED** [review]

**Review by yume-chan — APPROVED** [review]

**Comment by rom1v** [comment]
Thank you :+1: Merged: 2edc73e4b80d35acc594adef2d35d99e743befed

**Pull request closed without merging**